components/eam/tools/topo_tool/cube_to_target/cube_to_target.F90

```diff
@@ -841,17 +882,70 @@ program convterr
   sgh30_target = SQRT(sgh30_target)
   WRITE(*,*) "min/max of sgh_target     : ",MINVAL(sgh_target),MAXVAL(sgh_target)
   WRITE(*,*) "min/max of sgh30_target   : ",MINVAL(sgh30_target),MAXVAL(sgh30_target)
+
+  !-----------------------------------------------------------------------------
+  ! Orographic shape parameters
+  if (calc_orographic_shape_params) then
+
+    nvar_dirOA=2+1
```

Comment:
What are these? Some doc would be good here

Reply:
updated.